packages/custom-types/participant.d.ts

```diff
@@ -64,5 +64,5 @@ export interface ParticipantCollab extends CoreParticipant {
   // participant views un-released documents
   viewUnreleasedVersions: boolean
   // level of access for the participant
-  permission: CollaborativePermission
+  permission: number[]
```

Comment:
Yes, I see this change.  But, our _model_ is that the Game Designer selects a permission, and the role has all permission levels up to that one.

Reply:
sure @IanMayo, update it back to `CollaborativePermission ` and we should verify the admin form again to make sure the Select permission works correctly :D